test/asserts/load.js

```diff
@@ -17,6 +17,19 @@ QUnit.module("load", function () {
         })['catch'](JSZipTestUtils.assertNoError);
     });
 
+    JSZipTestUtils.testZipFile("Load files which shadow Object prototype methods", "ref/pollution.zip", function(assert, file) {
+        var done = assert.async();
+        assert.ok(typeof file === "string");
+        JSZip.loadAsync(file)
+        .then(function (zip) {
+            return zip.file("toString").async("string");
+        })
+        .then(function(result) {
+            assert.equal(result, "hello\n", "the zip was correctly read.");
+            done();
+        })['catch'](JSZipTestUtils.assertNoError);
+    });
```

Comment:
I tried removing the `Object.create(null)` fix, to see if this test failed and it didn't, so I don't think this is actually catching the issue?

Reply:
That's a good point - but I'm not entirely sure how to test this change because essentially this change is about making sure `toString` / `__proto__` / `__constructor__` etc are not available in the first place.

I guess I could add a test that shows that `toString` raises an error - but I'm not sure how useful that would be in the long run?

The reasoning for this test specifically was to show that files with names that shadow standard Object method names can still be accessed correctly